sdk/notificationhubs/notification-hubs/review/notification-hubs-api.api.md

```diff
@@ -11,7 +11,10 @@ import { OperationState } from '@azure/core-lro';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { PipelineRequest } from '@azure/core-rest-pipeline';
 import { PipelineResponse } from '@azure/core-rest-pipeline';
-import { SimplePollerLike } from '@azure/core-lro';
+import { PollerLike } from '@azure/core-lro';
+
+// @public
+export function beginGetNotificationDetails(context: NotificationHubsClientContext, notificationId: string, polledOperationOptions?: PolledOperationOptions): Promise<NotificationDetailsPoller>;
```

Comment:
This is the operation that will only be available to some SKUs but not to all correct? I was thinking, it may be a good idea to expose these operations in a separate subExport that makes it clear they are meant to be used for "non-basic" skus.

If we do this, I think it would be a good idea, instead of exposing an LRO for getting the notification details. Exposing a `submitNotification` that returns a PollerLike so that customers that have these Skus only need to call one operation instead of submit + check.